### PR TITLE
DEV-3517-new go exercise shoppinglistsort

### PR DESCRIPTION
**Pull request by tiagocollot**

### CODE REVIEW

**Review by jotapero — APPROVED** [review]
Everything in order

File: solutions/shoppinglistsort.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package solutions
+
+func ShoppingListSort(array[] string)[]string {
```
Comment (eslopfer) [inline solutions/shoppinglistsort.go:3]: It shouldn't be called array.
Also please run gofmt.

File: tests/shoppinglistsort_test/main.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	array := [][]string{
```
Comment (eslopfer) [inline tests/shoppinglistsort_test/main.go:11]: Name and indentations are not correct.

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: solutions/shoppinglistsort.go (hunk 1 above)
Comment (tiagocollot) [reply solutions/shoppinglistsort.go]: Name and indentation fixed!

Comment (tiagocollot) [reply solutions/shoppinglistsort.go]: Argument name and indentation fixed!

File: tests/shoppinglistsort_test/main.go (hunk 2 above)
Comment (tiagocollot) [reply tests/shoppinglistsort_test/main.go]: Variable name and indentation fixed!

**Comment by eslopfer** [comment]
The comments have been addressed, but I can't approve until you implement a robust way to handle the order when length is the same. Right now if the length is the same, you expect the elements to be sorted in the same way as in your solution, but in the description you say that the order doesn't matter.

**Review by eslopfer — APPROVED** [review]

**Pull request merged**